backend/frontend/views.py

```diff
@@ -672,6 +672,44 @@ def action_stop_card(request, card_id):
     )
 
 
+@login_required()
+def view_partial_users_list(request):
+    exclude = {
+        "is_superuser": True,
+    }
+    all_users = User.objects.filter(active=True).order_by("first_name", "last_name").exclude(**exclude)
+    total_user_count = all_users.count()
+
+    if request.method == "POST":
+        form = SimpleSearchForm(request.POST)
+
+        if form.is_valid():
+            search_term = form.cleaned_data["search_term"]
+
+            all_users = User.objects.filter(active=True).filter(
```

Comment:
Rather make a base query and then filter it instead of writing the same query twice. 

So:
```
all_users = User.objects.filter(.....
...
if form.is_valid()
     all_iusers = all_users.filter(...)
```

Also, why are you only filtering by istartswith? Why not icontains? Surely we dont need the people using this to remember how things start?

Also there might be multiple search terms. Eg: "Percival Rapha" in which case we would want to search for each word.